applications/nrf5340_audio/src/audio/streamctrl_broadcast_sink.c

```diff
@@ -584,17 +448,80 @@ static int zbus_subscribers_create(void)
 		return ret;
 	};
```

Comment:
```suggestion
	}
```

Reply:
I see the same in streamctrl_broadcast_source

Reply:
Changed in both files